src/activities/ActivityUsageEntity.js

```diff
@@ -104,4 +104,22 @@ export class ActivityUsageEntity extends Entity {
 		const fields = [{ name: 'dueDate', value: dateString }];
 		await performSirenAction(this._token, action, fields);
 	}
+
+	canEditDraft() {
+		return this._entity && this._entity.hasActionByName(Actions.activities.updateDraft);
+	}
+
+	async setDraftStatus(isDraft) {
```

Comment:
Some jsdocs would be great here

Reply:
Ah shit I haven't done that at all - I'll make a note to do so in the stuff I've added

Reply:
no worries.  It'd just be good to get the docs running so people could reference what the sdk can do without digging in code

Reply:
Ran the generation for the ActivityUsageEntity, let me know if there is other steps in the workflow here.

Reply:
Adding detail to the jsdoc for public methods would make it easier for consumers to reason  about the function call.  Something similar to https://github.com/BrightspaceHypermediaComponents/siren-sdk/blob/master/src/consortium/ConsortiumTokenEntity.js 

This is an example of how to use jsdocs in general https://devdocs.io/jsdoc/howto-es2015-classes